config.go

```diff
@@ -157,6 +157,15 @@ func getConfig() *types.Configuration {
 	v.SetDefault("Rabbitmq.Queue", "")
 	v.SetDefault("Rabbitmq.MinimumPriority", "")
 
+	v.SetDefault("Wavefront.EndpointType", "")
+	v.SetDefault("Wavefront.EndpointHost", "")
+	v.SetDefault("Wavefront.EndpointToken", "")
+	v.SetDefault("Wavefront.MetricName", "falco.alert")
+	v.SetDefault("Wavefront.MetricPort", 2878)
+	v.SetDefault("Wavefront.MinimumPriority", "")
+	v.SetDefault("Wavefront.FlushIntervalSecods", 1)
+	v.SetDefault("Wavefront.BatchSize", 10000)
```

Comment:
`MetricPort`, `FlushIntervalSecods` and `BatchSize` are not mentionned in README.